nd4j/nd4j-backends/nd4j-tests/src/test/java/org/nd4j/autodiff/samediff/listeners/ListenerTest.java

```diff
@@ -130,8 +130,8 @@ public void testListenerCalls(){
         SDVariable w = sd.var("w", Nd4j.rand(DataType.FLOAT, 4, 3));
         SDVariable b = sd.var("b", Nd4j.rand(DataType.FLOAT, 3));
         SDVariable z = in.mmul(w).add(b);
-        SDVariable softmax = sd.nn.softmax("softmax", z);
-        SDVariable loss = sd.loss.logLoss("loss" ,label, softmax);
+        SDVariable softmax = sd.nn.softmax("softmax", z, 0);
+        SDVariable loss = sd.loss.logLoss("loss" ,label, softmax, null, 0.0);
```

Comment:
Bad softmax dimension + logLoss should have default value (so (String, SDVariable, SDVariable) method exists)